### ISSUE TITLE
searching in calendar / tasks doesn't work in SOGo

### ISSUE THREAD

**Issue by efpec** [mailcow/mailcow-dockerized#4660]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/mailcow/mailcow-dockerized/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... I understand that not following the below instructions will result in immediate closure and/or deletion of my issue.
- [X] ... I have understood that this bug report is dedicated for bugs, and not for support-related inquiries.
- [X] ... I have understood that answers are voluntary and community-driven, and not commercial support.
- [X] ... I have verified that my issue has not been already answered in the past. I also checked previous [issues](https://github.com/mailcow/mailcow-dockerized/issues).

### Description

Searching for tasks and calendar entries doesn't work in SOGo.
Searching for mails or contacts works fine.

Tested with a new and fresh installed Mailcow with only some created testdata.

I asked in the community, another user confirmed the problem. 
https://community.mailcow.email/d/1711-suche-termine-aufgaben-in-sogo

### Logs

```bash
n.a.
```

### Steps to reproduce

1. Install new Mailcow
2. Create a user with some calendar entries
3. Open SOGo, search your calendar entry and find nothing ;-)

### System information

| Question | Answer |
| --- | --- |
| My operating system | I_DO_REPLY_HERE |
| Is Apparmor, SELinux or similar active? | I_DO_REPLY_HERE |
| Virtualization technology (KVM, VMware, Xen, etc - **LXC and OpenVZ are not supported** | I_DO_REPLY_HERE |
| Server/VM specifications (Memory, CPU Cores) | I_DO_REPLY_HERE |
| Docker version (`docker version`) | I_DO_REPLY_HERE |
| docker-compose version (`docker-compose version`) | I_DO_REPLY_HERE |
| mailcow version (```git describe --tags `git rev-list --tags --max-count=1` ```) | I_DO_REPLY_HERE |
| Reverse proxy (custom solution) | I_DO_REPLY_HERE |

Output of `git diff origin/master`, any other changes to the code? If so, **please post them**:
```
YOUR OUTPUT GOES HERE
```

All third-party firewalls and custom iptables rules are unsupported. **Please check the Docker docs about how to use Docker with your own ruleset**. Nevertheless, iptabels output can help us to help you:
iptables -L -vn:
```
YOUR OUTPUT GOES HERE
```

ip6tables -L -vn:
```
YOUR OUTPUT GOES HERE
```

iptables -L -vn -t nat:
```
YOUR OUTPUT GOES HERE
```

ip6tables -L -vn -t nat:
```
YOUR OUTPUT GOES HERE
```

DNS problems? Please run `docker exec -it $(docker ps -qf name=acme-mailcow) dig +short stackoverflow.com @172.22.1.254` (set the IP accordingly, if you changed the internal mailcow network) and post the output:
```
YOUR OUTPUT GOES HERE
```

**Comment by MAGICCC** [comment]
As seen in the forum, it should be fixed in SoGo 5.7.0, which will be in mailcow with the next release
https://github.com/mailcow/mailcow-dockerized/pull/4632

**Comment by efpec** [comment]
I hope so 😅
Can't wait for the new release 😉

**Comment by MAGICCC** [comment]
Hi @efpec 
Can you update to https://github.com/mailcow/mailcow-dockerized/releases/tag/2022-07 and check again?

**Comment by efpec** [comment]
Thank you! Solved in 2022-07 with SOGo 5.7.0 👍🏼
